packages/lit-dev-content/site/tutorials/content/pointer-events/03.md

```diff
@@ -0,0 +1,15 @@
+Pointer Capturing allows for a pointer to be retargeted to a chosen element, rather than the normal one set via hit testing.
+
+In practice, this means you can make mouse interactions behave like touch, and touch interactions behave like mouse.
+For completeness, touch pointers are considered "implicitly captured".
+
+**Pointer Capturing API**
+
+* To capture a pointer, you can use the `setPointerCapture(pointerId)` method on any HTML element.
```

Comment:
I suggest you refer to the code on the right to show rather than tell.

> to capture a pointer... we do this in... (this second part can even be a code snippet)